code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -123,11 +123,18 @@
 	list_reagents = list("epinephrine" = 30, "weak_omnizine" = 30, "salglu_solution" = 30)
 
 /obj/item/reagent_containers/hypospray/combat/nanites
-	desc = "A modified air-needle autoinjector for use in combat situations. Prefilled with expensive medical nanites for rapid healing."
+	desc = "A modified air-needle autoinjector for use in combat situations. Prefilled with powerful medical nanomachines, to keep your body in the fight."
 	icon_state = "nanites_hypo"
 	volume = 100
 	list_reagents = list("nanites" = 100)
 
+/obj/item/reagent_containers/hypospray/combat/syndicate_nanites
+	name = "medical nanite injector"
+	desc = "A modified air-needle autoinjector for use in combat situations. Prefilled with expensive medical nanites for rapid field stabilization."
```

Comment:
```suggestion
	desc = "A modified air-needle autoinjector for use in combat situations. Prefilled with expensive medical nanomachines for rapid field stabilization."
```
Other injector uses "nanomachines" instead of "nanites"